mac/sampleoutput.txt

```diff
@@ -0,0 +1,617 @@
+{
+ "count": 153,
```

Comment:
This file is just to serve as an example of what the script returns? Meaning we won't use the whole content?
If so, could we trim it down to a few lines instead?

Reply:
true